docs/conceptual/compiler-topics.md

```diff
@@ -0,0 +1,12 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="AMD ROCm documentation">
+  <meta name="keywords" content="documentation, guides, installation, compatibility, support,
+  reference, ROCm, AMD">
+</head>
+
+# Compiler topics
+
+* [Address sanitizer](./using-gpu-sanitizer.md)
```

Comment:
AddressSanitizer rather than Address Sanitizer for consistency

Reply:
Fixed